functions-and-scope.js

```diff
@@ -70,6 +113,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
+let highestGrade = 0;
```

Comment:
Je kan hier beter niet 0 maar de eerste waarde uit de array pakken. Het zal hier altijd goed gaan, omdat in de praktijk cijfers niet lager dan 0 kunnen zijn. Maar stel je voor dat je een array aanlevert van [-1, -2, -5, -3], dan krijg je 0 terug wat dus niet klopt.